public/img/logos/EpicGames.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 18.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Just a self-reminder that we should run all these images though svgo optimizer/minification after we merge in the logos.

Reply:
Did you modify these image assets at all, or was this a pure copy-pasta from the **blurts-addon** repo images?